src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -1024,9 +1066,8 @@ okapiHeaders, null, handlePreviousResponse(false, false, false,
           }
 
           if(!aRequestHasFailed[0]){
-            asyncResultHandler.handle(Future.succeededFuture(
-              PostBlUsersLoginResponse.respond201WithApplicationJson(cu,
-                PostBlUsersLoginResponse.headersFor201().withXOkapiToken(token))));
+            javax.ws.rs.core.Response r = respond.apply(loginResponse, cu);
```

Comment:
Maybe it's better to replace the line as: `var response = respond.apply(loginResponse, cu);`

Reply:
Changed